src/main/java/ru/practicum/shareit/booking/mapper/BookingMapper.java

```diff
@@ -0,0 +1,19 @@
+package ru.practicum.shareit.booking.mapper;
+
+import ru.practicum.shareit.booking.dto.BookingDto;
+import ru.practicum.shareit.booking.model.Booking;
+
+public final class BookingMapper {
+
+    public static BookingDto toBookingDto(Booking booking) {
+        BookingDto dto = BookingDto.builder()
+                .id(booking.getId())
+                .start(booking.getStart())
+                .end(booking.getEnd())
+                .item(booking.getItem())
+                .booker(booking.getBooker())
+                .status(booking.getStatus())
+                .build();
+        return dto;
```

Comment:
можно немного сократить
```suggestion
        return BookingDto.builder()
                .id(booking.getId())
                .start(booking.getStart())
                .end(booking.getEnd())
                .item(booking.getItem())
                .booker(booking.getBooker())
                .status(booking.getStatus())
                .build();
```

Reply:
сделано